packages/devextreme/testing/tests/DevExpress.ui.widgets/splitter.tests.js

```diff
@@ -2116,6 +2117,98 @@ QUnit.module('Events', moduleConfig, () => {
 });
 
 QUnit.module('Nested Splitters', moduleConfig, () => {
+    function getNestedSplitter(splitterElement) {
+        return splitterElement.find(`.${SPLITTER_CLASS}`);
+    }
+
+    [
+        { propertyName: 'allowKeyboardNavigation', propertyValue: false },
```

Comment:
There are no tests for testing all events. onItemExpanded, onItemCollapsed for example.